### DISCUSSION TITLE
Managing 10+ whatsapp numbers

### DISCUSSION THREAD

**Discussion by evollution** [devlikeapro/waha#560] [Q&A]
Hello, what app do you recommend to manage 10+ whatsapp numbers, i mean when get customer messages to see the message, no matter from which phone we get it

**Comment by devlikepro** [comment]
Hi! 
Do you mean like a single WhatsApp Web for multiple wa numbers, something like multi agent? 
We don't have the solution for this right now, but we're working on **WAHA + ChatWoot** integration through n8n (the similar we did with https://waha.devlike.pro/whatsapp-typebot/), approximately will have it in the start of October :crossed_fingers:  

Try searching "multi agent whatsapp" in google, there's many SaaS for that already (and few of them build with WAHA, I believe :rofl: )

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by FaheemAlvii** [reply]
> Hi! Do you mean like a single WhatsApp Web for multiple wa numbers, something like multi agent? We don't have the solution for this right now, but we're working on **WAHA + ChatWoot** integration through n8n (the similar we did with https://waha.devlike.pro/whatsapp-typebot/), approximately will have it in the start of October 🤞
> 
> Try searching "multi agent whatsapp" in google, there's many SaaS for that already (and few of them build with WAHA, I believe 🤣 )
> 
> [![patron:PRO](https://camo.githubusercontent.com/4f70c3a9ee2450c288faea5e1a1b7ebd0ffc1571450a59676b300ffb32b32537/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f706174726f6e2d50524f2d313838613432)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

Really looking forward to this one.. it would be really helpful!

**Reply by devlikepro** [reply]
https://waha.devlike.pro/docs/integrations/chatwoot/
https://waha-n8n-templates.devlike.pro/chatwoot/

It's a bit hard to setup for now, like we'll add it as an built-in feature in the next year :crossed_fingers: 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
